Library/Formula/dnscrypt-proxy.rb

```diff
@@ -26,6 +27,26 @@ class DnscryptProxy < Formula
 
   depends_on "libsodium"
 
+  def updateresolvers; <<-EOS.undent
+    #!/bin/sh
+    RESOLVERS_UPDATES_BASE_URL=https://download.dnscrypt.org/dnscrypt-proxy
+    RESOLVERS_LIST_BASE_DIR=#{share}/dnscrypt-proxy
```

Comment:
This would just be `pkgshare`, which is shorthand for `#{share}/dnscrypt-proxy`.